hack/platform/partials/main.go

```diff
@@ -896,16 +907,63 @@ spec:
 	})
 
 	util.DefaultRequire = false
+	schema := &jsonschema.Schema{}
+	schemaBytes, err := os.ReadFile(jsonSchemaPath)
+	if err != nil {
+		panic(err)
+	}
+	err = json.Unmarshal(schemaBytes, schema)
+	if err != nil {
+		panic(err)
+	}
+	externalProperty, ok := schema.Properties.Get("external")
```

Comment:
More descriptive error messages ideas

```suggestion
	schema := &jsonschema.Schema{}
	schemaBytes, err := os.ReadFile(jsonSchemaPath)
	if err != nil {
		panic(fmt.Errorf("failed to read schema file %q: %w", jsonSchemaPath, err))
	}
	err = json.Unmarshal(schemaBytes, schema)
	if err != nil {
		panic(fmt.Errorf("failed to parse JSON schema from %q: %w", jsonSchemaPath, err))
	}
	externalProperty, ok := schema.Properties.Get("external")
```

Reply:
good point, fixed